src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+controller machine directly via the cloud provider, without cleaning up any
+machines that may be running in workload models.
+
+The `show-` prefix is used to drill down into an object to reveal details. It
+is akin to the verb "describe".
+
+There are command aliases, that typically employ a prefix, that point to their
+corresponding commands. This practice is for simplifying the life of the
+operator. The `list-` alias/prefix is a good example, where `list-clouds`
+points to `clouds`.
+
+The list of aliases can be obtained in this way:
+
+```bash
+juju help commands | grep Alias
+```
+
+## Fundamental commands
+
+Juju makes available to the operator a wide variety of commands. In this
+section, we'll cover the basic ones. 
+
+### Listing initial information
+
+List the controller:
+
+```bash
+juju controllers
+```
+
+This will return a list of all the controllers known to your Juju client.
+Below, we have a controller named 'lxd':
+
+```no-highlight
+Controller  Model    User   Access     Cloud/Region         Models  Machines    HA  Version
+lxd*        default  admin  superuser  localhost/localhost       2         1  none  2.5.0
+```
+
+A newly-created controller has two models: The 'controller' model, which should
+be used only for internal Juju management, and a 'default' model, which is
+ready for actual use.
+
+Confirm the above by listing all the models in the currently active controller:
+
+```bash
+juju models
+```
+
+Our example's output:
+
+```no-highlight
+Controller: lxd
+
+Model       Cloud/Region         Status     Machines  Access  Last connection
+controller  localhost/localhost  available         1  admin   just now
+default*    localhost/localhost  available         2  admin   2019-02-02
+```
+
+To see the currently active controller, model, and user:
+
+```bash
+juju whoami
+```
+
+Our example:
+
+```no-highlight
+Controller:  lxd
+Model:       default
+User:        admin
+```
+
+### Deploying an application
+
+Applications are contained within models and are installed via *charms*. By
+default, charms are downloaded from the online [Charm Store][charm-store]
+during deployment.
+
+To deploy a charm, such as 'redis', in the currently active model:
+
+```bash
+juju deploy redis
+```
+
+This causes a similarly-named application ("redis") to be installed on a
+newly-created machine within the backing cloud.
+
+### Adding a model
+
+To create a model, named 'alpha', in the currently active controller:
+
+```bash
+juju add-model alpha
+```
+
+By default, when a model is created the currently active model becomes the new
+model.
+
+### Changing models
+
+To manually change to a different model, say the original 'default' model:
+
+```bash
+juju switch default
+```
+
+### Adding a machine
+
+To request that machine be created that is devoid of an application:
+
+```bash
+juju add-machine -m alpha -n 2
+```
+
+Here we've used the `-m` option to explicitly select a model (i.e. we don't
```

Comment:
we also add in the -n flag without noting it in any way. Is it necessary? If so, should we call out that we're in fact adding two here?

Reply:
OK, I missed that.